spark3/src/test/java/org/apache/iceberg/actions/TestCreateActions.java

```diff
@@ -296,14 +406,25 @@ private void createSourceTable(String createStatement, String tableName)
   private void assertMigratedFileCount(CreateAction migrateAction, String source, String dest)
       throws NoSuchTableException, NoSuchDatabaseException, ParseException {
     CatalogTable sourceTable = loadSessionTable(source);
-    Path sourceDir = Paths.get(sourceTable.location());
-    List<File> files = FileUtils.listFiles(sourceDir.toFile(), TrueFileFilter.INSTANCE, TrueFileFilter.INSTANCE)
-        .stream()
-        .filter(file -> !file.toString().endsWith("crc") && !file.toString().contains("_SUCCESS"))
-        .collect(Collectors.toList());
-    long expectedMigratedFiles = files.size();
-    List<Row> expected = spark.table(source).collectAsList();
+    List<URI> uris;
+    if (sourceTable.partitionColumnNames().size() == 0) {
+      uris = new ArrayList<>();
+      uris.add(sourceTable.location());
+    } else {
+      Seq<CatalogTablePartition> catalogTablePartitionSeq =
+              spark.sessionState().catalog().listPartitions(sourceTable.identifier(), Option.apply(null));
```

Comment:
should be formatted with 4 spaces instead of 8.

Reply:
Fixed